<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen_filelist" />
    <addaction name="actionOpen_own_filelist" />
    <addaction name="actionRefresh_own_filelist" />
    <addaction name="actionOpen_Downloads_Folder" />
    <addaction name="separator" />
    <addaction name="actionQuick_Connect" />
    <addaction name="actionFollow_last_redirect" />
    <addaction name="actionReconnect" />
    <addaction name="separator" />
    <addaction name="actionSettings" />
    <addaction name="actionGet_TTH_for_file" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionPublic_Hubs" />
    <addaction name="actionFavorite_Hubs" />
    <addaction name="separator" />
    <addaction name="actionFavorite_Users" />
    <addaction name="actionIgnored_Users" />
    <addaction name="separator" />
    <addaction name="actionSearch" />
    <addaction name="actionADL_Search" />
    <addaction name="actionSearch_Spy" />
    <addaction name="separator" />
    <addaction name="actionIndexing_Progress" />
    <addaction name="separator" />
    <addaction name="actionToolBar" />
    <addaction name="actionStatusBar" />
    <addaction name="actionTransfers" />
    <addaction name="actionTabBar" />
   </widget>
   <widget class="QMenu" name="menuTransfers" >
    <property name="title" >
     <string>Transfers</string>
    </property>
    <addaction name="actionDownload_Queue" />
    <addaction name="actionFinished_downloads" />
    <addaction name="separator" />
    <addaction name="actionFinished_Uploads" />
   </widget>
   <widget class="QMenu" name="menuWindow" >
    <property name="title" >
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="separator" />
    <addaction name="actionDonate" />
    <addaction name="actionHomepage" />
    <addaction name="separator" />
    <addaction name="aboutqtact" />
   </widget>
   <addaction name="menuFiles" />
   <addaction name="menuView" />
   <addaction name="menuTransfers" />
   <addaction name="menuWindow" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="sizeGripEnabled" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="WindowToolBar" >
   <property name="font" >
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="windowTitle" >
    <string>WindowToolBar</string>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <property name="allowedAreas" >
    <set>Qt::BottomToolBarArea|Qt::NoToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="TransDock" >
   <property name="features" >
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::NoDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Transfers</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QColumnView" name="columnView" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Main</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="statusTip" >
    <string>Quit from StilDC++ application</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_filelist" >
   <property name="text" >
    <string>Open filelist</string>
   </property>
   <property name="statusTip" >
    <string>Open Your Own FileList</string>
   </property>
  </action>
  <action name="actionOpen_own_filelist" >
   <property name="text" >
    <string>Open own filelist</string>
   </property>
   <property name="statusTip" >
    <string>Open FileList From File...</string>
   </property>
  </action>
  <action name="actionRefresh_own_filelist" >
   <property name="text" >
    <string>Refresh own filelist</string>
   </property>
   <property name="statusTip" >
    <string>Refresh Your FileList</string>
   </property>
  </action>
  <action name="actionOpen_Downloads_Folder" >
   <property name="text" >
    <string>Open Downloads Folder</string>
   </property>
   <property name="statusTip" >
    <string>Open Downloads Folder in FileManager</string>
   </property>
  </action>
  <action name="actionQuick_Connect" >
   <property name="text" >
    <string>Quick Connect</string>
   </property>
   <property name="statusTip" >
    <string>Quick Connect to Hub</string>
   </property>
  </action>
  <action name="actionFollow_last_redirect" >
   <property name="text" >
    <string>Follow last redirect</string>
   </property>
  </action>
  <action name="actionReconnect" >
   <property name="text" >
    <string>Reconnect</string>
   </property>
   <property name="statusTip" >
    <string>Reconnect to Hub</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="text" >
    <string>Settings</string>
   </property>
   <property name="statusTip" >
    <string>Configure StilDC++ client</string>
   </property>
  </action>
  <action name="actionGet_TTH_for_file" >
   <property name="text" >
    <string>Get TTH for file...</string>
   </property>
   <property name="statusTip" >
    <string>Calculate TTH for custom file...</string>
   </property>
  </action>
  <action name="actionPublic_Hubs" >
   <property name="text" >
    <string>Public Hubs</string>
   </property>
  </action>
  <action name="actionFavorite_Hubs" >
   <property name="text" >
    <string>Favorite Hubs</string>
   </property>
  </action>
  <action name="actionFavorite_Users" >
   <property name="text" >
    <string>Favorite Users</string>
   </property>
  </action>
  <action name="actionIgnored_Users" >
   <property name="text" >
    <string>Ignored Users</string>
   </property>
  </action>
  <action name="actionSearch" >
   <property name="text" >
    <string>Search</string>
   </property>
  </action>
  <action name="actionADL_Search" >
   <property name="text" >
    <string>ADL Search</string>
   </property>
  </action>
  <action name="actionSearch_Spy" >
   <property name="text" >
    <string>Search Spy</string>
   </property>
  </action>
  <action name="actionNotepad" >
   <property name="text" >
    <string>Notepad</string>
   </property>
  </action>
  <action name="actionIndexing_Progress" >
   <property name="text" >
    <string>Indexing Progress</string>
   </property>
  </action>
  <action name="actionToolBar" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>ToolBar</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionStatusBar" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>StatusBar</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionTransfers" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Transfers</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionDownload_Queue" >
   <property name="text" >
    <string>Download Queue</string>
   </property>
  </action>
  <action name="actionFinished_downloads" >
   <property name="text" >
    <string>Finished downloads</string>
   </property>
  </action>
  <action name="actionFinished_Uploads" >
   <property name="text" >
    <string>Finished Uploads</string>
   </property>
  </action>
  <action name="actionHomepage" >
   <property name="text" >
    <string>Homepage</string>
   </property>
  </action>
  <action name="actionDonate" >
   <property name="text" >
    <string>Donate</string>
   </property>
  </action>
  <action name="aboutqtact" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionTabBar" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Window Bar</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
